templates/company-board/contracts/CompanyBoardTemplate.sol

```diff
@@ -8,29 +8,19 @@ contract CompanyBoardTemplate is BaseTemplate {
     string constant private ERROR_MISSING_BOARD_MEMBERS = "COMPANY_MISSING_BOARD_MEMBERS";
     string constant private ERROR_MISSING_SHARE_MEMBERS = "COMPANY_MISSING_SHARE_MEMBERS";
     string constant private ERROR_BAD_HOLDERS_STAKES_LEN = "COMPANY_BAD_HOLDERS_STAKES_LEN";
-
-    uint64 constant private ONE_PCT = uint64(1e16);
-    uint64 constant private FINANCE_PERIOD = uint64(30 days);
+    string constant private ERROR_BAD_VOTE_SETTINGS = "COMPANY_BAD_VOTE_SETTINGS";
 
     bool constant private BOARD_TRANSFERABLE = false;
-    string constant private BOARD_TOKEN_NAME = "Board Token";
+    string constant private BOARD_TOKEN_NAME = "Board Token";	
```

Comment:
Note that there's some extra whitespace lingering in most of the contract files.

(LMK if your editor doesn't have an easy way to remove them, I use [this vim command](https://github.com/bronson/vim-trailing-whitespace) to fix it automatically)

Reply:
Oops! Thanks for catching that. I believe that @facuspagnuolo already fixed this in https://github.com/aragon/dao-templates/pull/123

I was planning on installing a linter to avoid this sort of stuff, but didn't find the time last week. Anyways, I installed that plugin for the mean time. Love it :D